data_processors/pipeline/domain/event/wrsc.py

```diff
@@ -0,0 +1,35 @@
+# generated by datamodel-codegen:
+#   filename:  WorkflowRunStateChange.schema.json
+#   timestamp: 2023-09-12T07:58:42+00:00
+
+from __future__ import annotations
+
+from datetime import datetime
+from typing import Optional
+
+from pydantic import BaseModel, Field, AliasChoices
+
+
+class WorkflowRunStateChange(BaseModel):
+    portal_run_id: str
+    type_name: str
+    version: str
+    output: str
```

Comment:
No input?
Because it's not used for initial workflow creation, i.e. only follow-on events?